tests/runner.py

```diff
@@ -1571,12 +1571,7 @@ def reftest(self, expected, manually_trigger=False):
       }
 ''' % (reporting.read(), basename, int(manually_trigger)))
 
-  def check_btest_skip(self, args):
-    if self.is_wasm_backend() and 'WASM=0' in args:
-      self.skipTest('wasm2js does not yet support threads, dynamic linking, and some other features; skip browser testing for now')
-
```

Comment:
Oh nice, a bunch of new tests pass with that? But they weren't minimal_runtime tests, were they, so why do they pass now?

Reply:
I went through all tests and manually skipped the ones that did not work with this, or fixed them.

The reason for that is this skip was too broad as it prevented me to do wasm2js checks in browser.